### ISSUE TITLE
Get azure subscriptions API is failing across ADS

### ISSUE THREAD

**Issue by aasimkhan30** [microsoft/azuredatastudio#15391]
It can be reproed on the latest ads insiders Version: 1.29.0 and the may RC1 release. 
 
Steps to Reproduce:

1. Go to any ads component that fetches azure subscriptions (Browse tab in connection, Azure pane, SQL Migration extension).
2. The loading of subscriptions fail.

 
![image](https://user-images.githubusercontent.com/6816294/117704673-f7b92100-b17f-11eb-9b2b-42acac640036.png)

This is cause by this PR https://github.com/microsoft/azuredatastudio/pull/15339

This has to do with the minimization that we do to optimize our final builds. The packages that I updated use a version node-fetch with a bug in it. node-fetch/node-fetch#667.

Basically, it uses constructor.name to validate the object at runtime. However, due to minimization we lose the original object name and thereby failing this check.

**Comment by kburtram** [comment]
Closing since fix is merged to release branch.
